frontend/src/employee-frontend/components/Header.tsx

```diff
@@ -208,7 +208,7 @@ const Header = React.memo(function Header({ location }: RouteComponentProps) {
               </RequireRole>
 
               {featureFlags.messaging &&
-                (featureFlags.experimental?.mobileDailyNotes ||
+                (featureFlags.experimental?.employeeMobile ||
                   hasPilotAccess) && (
```

Comment:
You probably forgot the change this conditional, just `hasPilotAccess` should be enough.

Reply:
Yeah, actually tried a slightly different approach there now.